packages/tabs/src/TabsContext.tsx

```diff
@@ -16,12 +27,10 @@ export const useTabContext = () => {
   return context
 }
 
-export const TabsProvider = (props: TabsProviderProps) => {
-  const { onOpen, children } = props
-
-  const value = {
-    onOpen,
-  }
-
-  return <TabsContext.Provider value={value}>{children}</TabsContext.Provider>
+export const TabsProvider = (
```

Comment:
? I'm using it now in Content, Tabs and Sidebar.

Reply:
@soofstad  the `TabsContext`  is only used here as @ingeridhellen mentioned.

Reply:
All right. If it makes sense to have a context just for this plugin thats fine.
Just a heads up to Ingerid that the context that was there, no longer has the original justification.

IMO, I think the context/provider can be removed for more readability. We'r only passing the context props one layer anyway.

Reply:
Yeah, I wouldn't have created a new context for this, but since it was already there... I agree readability with context is a bit lower than without, but who knows what functionality we might need to add in the future here, so I vote we keep it a while longer.